src/views/Registration/RegistrationDefineBusiness.vue

```diff
@@ -167,8 +181,10 @@ export default class RegistrationDefineBusiness extends Mixins(CommonMixin) {
   @Action setRegistrationDefineBusinessValid!: ActionBindingIF
   @Action setFolioNumber!: ActionBindingIF
   @Action setRegistrationBusinessNumber!: ActionBindingIF
+  @Action setRegistrationBusinessTypeGP!: ActionBindingIF
```

Comment:
This is a new action, right? You'll need to add it, and its mutation, and check in those file.

Reply:
I've added the mutation and its subsequent contents. Please review the code.

Reply:
Looks good.